stl/inc/xmemory

```diff
@@ -2174,6 +2174,35 @@ _NODISCARD constexpr bool _Allocators_equal(const _Alloc& _Lhs, const _Alloc& _R
     }
 }
 
+#if _HAS_CXX23
+template <class _Ty, class = void>
+inline constexpr bool _Has_member_from_primary = false;
+template <class _Ty>
+inline constexpr bool _Has_member_from_primary<_Ty, void_t<typename _Ty::_From_primary>> = true;
+
+// Avoid using allocate_at_least when the allocator publicly derives from std::allocator:
+// "old" allocators might override allocate and deallocate but fail to override allocate_at_least.

```

Comment:
Terminology nitpick: only `virtual` functions can be overridden. These are being hidden.